### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by Zelderon7**
Merging master and homework

### CODE REVIEW

File: src/main/java/org/example/hw2.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package org.example;
+
+import java.util.Arrays;
+
+public class hw2 {
+
+
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/hw2.java:7]: Remove unnecessary line.

File: src/main/java/org/example/hw2.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    public static void main(String[] args) {
+
+        System.out.println(sumArr(new int[] {2, 34, 5, 67, 78, 89, 8, 3}));
+    }
+    static int sumArr(int[] arr){
+        return Arrays.stream(arr).sum();
+    }
+
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/hw2.java:15]: Remove unnecessary line.

File: src/main/java/org/example/zad2.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class zad2 {
+    static Scanner in = new Scanner(System.in);
+
+    public static void main(String[] args) {
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad2.java:8]: Add new line after the method. After: public static void main(String[] args) {.

File: src/main/java/org/example/zad2.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println(String.join("\n", in.next().split("")));
+
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad2.java:10]: Remove unnecessary line.

File: src/main/java/org/example/zad3.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class zad3 {
+
+    static Scanner in = new Scanner(System.in);
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad3.java:7]: Add new line after the variable. Add new line after "static Scanner in = new Scanner(System.in);".

File: src/main/java/org/example/zad5.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
+package org.example;
+
+import java.util.Arrays;
+
+public class zad5 {
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad5.java:5]: Smart solution. :)

File: src/main/java/org/example/zad3.java [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    public static void main(String[] args) {
+
+        System.out.print("Enter weight: ");
+        int w = getInt(50, 500);
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad3.java:11]: The variable and method are not explaining what they are doing. Be more descriptive.

File: src/main/java/org/example/zad3.java [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+                    break;
+            }
+        }
+
+    }
+
+    private static int getInt(int min, int max){
+        Scanner in = new Scanner(System.in);
+        int a = -1;
+        while (a < min || a >= max){
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad3.java:46]: Add space between ) and {.

File: src/main/java/org/example/zad3.java [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+                default:
+                    w--;
+                    break;
+            }
+        }
+
+    }
+
+    private static int getInt(int min, int max){
```
Comment (Evgeni-Georgiev) [inline src/main/java/org/example/zad3.java:43]: Add space between ) and {.

**Review by Evgeni-Georgiev — CHANGES_REQUESTED** [review]
Good job.

**Review by intro-java — APPROVED** [review]

**Review by Evgeni-Georgiev — APPROVED** [review]

**Pull request merged**